src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py

```diff
@@ -195,6 +200,12 @@ def readFWJRForErrors(self, jobList):
         cooloffJobs = []
         passJobs = []
         exhaustJobs = []
+
+        if self.reqAuxDB is None:
+            exitCodesNoRetry = self.config.ErrorHandler.failureExitCodes
```

Comment:
Hi Alan, T0 was always overwriting that. 

https://github.com/dmwm/T0/blob/master/bin/tier0-mod-config#L95

But if T0 never used it I would prefer to make [] here. Although this is more flexible.